strawberryfields/ops.py

```diff
@@ -1424,16 +1424,18 @@ class PassiveChannel(Channel):
         Acts the following transformation on the state:
 
         .. math::
-            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}j
+            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}_j
+
```

Comment:
For some reason, I can't see what changed here?

Reply:
i think all i changed was just the underscore on the final a^{\dagger}